content/teams/The Bug Slayers/index.md

```diff
@@ -1,55 +1,68 @@
 ---
-type: "team"                                                        
+type: "team"                   
 title: The Bug Slayer
 name: "The Bug Slayer"
-Team-Quote: "We Just Came here for Candy"
-Team_Tech_Stack: "Bootstrap,HTML,CSS,JavaScript,Firebase,Github"
+github-link: "https://github.com/Amik-Sen/CODE-CAMP-2020.git"
+description: "Hello we are Web Developers"
+Team-Quote: "We just came here for Candy"
+Team_Tech_Stack: "BootStrap, Firebase, HTML, CSS, Javascript"
 Application_Type: "Web Application"
-No_Of_Member: "2"
+No_Of_Member: ""
 ---
 
-Hey ! We are Team The Bug Slayer. We are here to win. and we also love open source.
+Hey ! We are Team The Bug Slayer. We just came here for Candy and we also love open source.
 
 ---
 
 ## Project's Overview
 
 _**Theme-Name**_ : AtmaNirbhar Bharat
 
-_**Application-Type**_ :   Web Application
+_**Application-Type**_ :   Web Application 
+
+_**Idea-Description**_ :   
+
+_**Tech-Stack-Used**_ :   
+
+_**GitHub-Link**_ :   
+
+_**Hosted-Link**_ :    
```

Comment:
can you Mention more details to these. Also leave Hosted Link : write N/A for now